README.md

```diff
@@ -86,23 +79,23 @@ let client = new TChannel();
 
 // Create the client subchannel that makes requests.
 let clientSubChannel = client.makeSubChannel({
-    serviceName: 'server',
-    peers: ['127.0.0.1:4040']
+  serviceName: 'server',
+  peers: ['127.0.0.1:4040'],
 });
 
 let encodedThriftChannel = TChannelAsThrift({
-    channel: clientSubChannel,
-    entryPoint: path.join(__dirname, 'thrift', 'echo.thrift') // file path to a thrift file
+  channel: clientSubChannel,
+  entryPoint: path.join(__dirname, 'thrift', 'echo.thrift'), // file path to a thrift file
```

Comment:
why comma?

Reply:
Hi Yuri, as discussed in #200, I followed [Jaeger UI's setup for Prettier](https://github.com/jaegertracing/jaeger-ui/blob/b7a3e74b6bd53f110755f2b4be76f3019a29de5e/package.json#L115) and they've got the trailing comma set to 'es5'.